src/transaction/log_checkpoint_info.hpp

```diff
@@ -17,8 +17,12 @@
  */
 
 #include "log_lsa.hpp"
-#include "log_record.hpp"
+#include "log_system_tran.hpp"
 #include "packable_object.hpp"
+#include "page_buffer.h"
+#include "system_parameter.h"
```

Comment:
I don't think all includes are needed. log_system_tran.hpp, page_buffer.h and system_parameter.h shouldn't be required in this header.

Reply:
log_system_trans is required for the function definitions of load_checkpoint_trans and load_checkpoint_topop, the rest have been moved